package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "eslint-config-digitalbazaar",
+  "version": "1.0.0",
+  "description": "basic eslint config for digital bazaar projects",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/digitalbazaar/eslint-config-digitalbazaar.git"
```

Comment:
```suggestion
    "url": "https://github.com/digitalbazaar/eslint-config-digitalbazaar.git"
```

Reply:
npm auto-generates this with git+ why remove it?

Reply:
Our publish script uses the URL to checkout the repo.  https://github.com/digitalbazaar/publish-script

Typically, we are using package files from other repos as a template, so it has not been a pain point.  For today, we'll need to use just HTTPs and feel welcome to put in an issue at publish-script to support `git+https` urls